boards/posix/native_posix/Kconfig.defconfig

```diff
@@ -103,3 +103,14 @@ config USB_NATIVE_POSIX
 	default y
 
 endif # USB
+
+# i386 assembly code used in MBEDTLS does not compile with speed optimization
+# if address sanitizer is enabled, as such switch default optimization level
+# to speed
```

Comment:
This comment doesn't make sense.

It says "We can't use X when we do Y, therefore do Y."

Reply:
Oops comment should read `i386 assembly code used in MBEDTLS does not compile with size optimization ...`